roles/certificate-acl/templates/certificate_acl_deployment.yaml.j2

```diff
@@ -0,0 +1,69 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    name: certificate-acl
+    app: dojot
+  name: certificate-acl
+  namespace: {{ dojot_namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: dojot
+      name: certificate-acl
+  template:
+    metadata:
+      labels:
+        name: certificate-acl
+        app: dojot
+    spec:
+      restartPolicy: Always
+      securityContext:
```

Comment:
Is this really used? Do you know what it means?

Reply:
restartPolicy = only refers to restarts of the containers by the kubelet on the same node. 
securityContext = To specify security settings for a Pod, include the securityContext field in the Pod specification.
Since fsGroup field is specified, all processes of the container are also part of the supplementary group ID